packs/data/spells.db/telekinetic-rend.json

```
@@ -67,6 +45,61 @@
         "materials": {
             "value": ""
         },
+        "overlays": {
+            "CufbZVoGmuZokCjW": {
+                "_id": "CufbZVoGmuZokCjW",
+                "name": "Telekinetic Rend (Bludgeoning)",
+                "overlayType": "override",
+                "sort": 1,
+                "system": {
+                    "damage": {
+                        "value": {
+                            "Q85SvGoxCoIiB0OS": {
+                                "applyMod": false,
+                                "type": {
+                                    "categories": [],
+                                    "subtype": "",
+                                    "value": "bludgeoning"
+                                },
+                                "value": "1d6"
+                            }
+                        }
+                    },
+                    "heightening": {
+                        "damage": {
+                            "Q85SvGoxCoIiB0OS": "1d6"
+                        }
+                    }
+                }
+            },
+            "rBTunIUathAlk04R": {
+                "_id": "rBTunIUathAlk04R",
+                "name": "Telekinetic Rend (Slashing)",
+                "overlayType": "override",
+                "sort": 2,
+                "system": {
+                    "damage": {
+                        "value": {
+                            "VSfHXFSJQGTzmS13": {
+                                "applyMod": false,
+                                "type": {
+                                    "categories": [],
+                                    "subtype": "",
+                                    "value": "slashing"
+                                },
+                                "value": "1d6"
+                            }
+                        }
+                    },
+                    "heightening": {
+                        "damage": {
+                            "VSfHXFSJQGTzmS13": "1d6",
+                            "fxdjllexjn5g19ig": "1d8"
```

```suggestion
                            "VSfHXFSJQGTzmS13": "1d6"
```
Something is weird about the spell variants, that ID doesn't even exist on the item.